### PR TITLE
fix sorting issue of pages and fix UI issue of process report page

### PR DESCRIPTION
**Pull request by mubeenmughal**
fix sorting issue of pages and fix UI issue of process report page

### CODE REVIEW

File: src/components/newSidebar/manageFile/ManageFile.jsx [hunk 1]
```diff
@@ -214,6 +214,13 @@ const ManageFile = () => {
     );
   }, [itemsCountToDisplay]);
 
+  console.log("allDocumentsArray", allDocumentsArray)
+
+
+  console.log("allWorkflowsArray", allWorkflowsArray)
+  console.log("allTemplatesArray", allTemplatesArray)
+  console.log("allProcesses", allProcesses)
+
```
Comment (WawinyEdwin) [inline src/components/newSidebar/manageFile/ManageFile.jsx:223]: Unnecessary logs should not go to production

File: src/components/newSidebar/reports/Reports.jsx [hunk 2]
```diff
@@ -32,6 +32,10 @@ const Reports = () => {
   );
   const { allProcesses, themeColor } = useSelector((state) => state.app);
 
+  console.log("allProcesses", allProcesses.filter((item) => item.processing_state === 'draft'))
+  console.log("allProcessesreport", allProcesses.filter((item) => item.processing_state === 'finalized'))
+  console.log("allProcessesreportprocessing", allProcesses.filter((item) => item.processing_state === 'processing').sort((a, b) => new Date(b?.created_on || b?.created_at) - new Date(a?.created_on || a?.created_at)))
+
```
Comment (WawinyEdwin) [inline src/components/newSidebar/reports/Reports.jsx:38]: Not required

File: src/pages/Folders/modals/FoldersModal.jsx [hunk 3]
```diff
@@ -285,6 +286,14 @@ const FoldersModal = () => {
 
   useEffect(() => {
     if (allDocuments)
+      console.log("allDocuments", allDocuments)
+      console.log("allDocuments", allDocuments
```
Comment (WawinyEdwin) [inline src/pages/Folders/modals/FoldersModal.jsx:290]: not required

File: src/pages/Folders/modals/FoldersModal.jsx [hunk 4]
```diff
@@ -655,6 +665,10 @@ const SelectInput = ({
   const superContainerRef = useRef(null);
   const containerRef = useRef(null);
 
+  console.log("selDocs", selDocs)
+  console.log("selTemps", selTemps)
+  console.log("list", list)
+
```
Comment (WawinyEdwin) [inline src/pages/Folders/modals/FoldersModal.jsx:671]: not required

File: src/pages/Folders/modals/FoldersModal.jsx [hunk 5]
```diff
@@ -306,9 +315,10 @@ const FoldersModal = () => {
   }, [allDocuments, allTemplates]);
 
   useEffect(() => {
+    console.log(docsList)
     const start = (docsListCurrentPage - 1) * itemsPerPage;
     const end = docsListCurrentPage * itemsPerPage;
-
+    console.log("docsList.slice(start, end)",docsList.slice(start, end))
```
Comment (WawinyEdwin) [inline src/pages/Folders/modals/FoldersModal.jsx:321]: not required

File: src/pages/Folders/modals/FoldersModal.jsx [hunk 6]
```diff
@@ -306,9 +315,10 @@ const FoldersModal = () => {
   }, [allDocuments, allTemplates]);
 
   useEffect(() => {
+    console.log(docsList)
```
Comment (WawinyEdwin) [inline src/pages/Folders/modals/FoldersModal.jsx:318]: not required

**Review by WawinyEdwin — CHANGES_REQUESTED** [review]

File: src/components/newSidebar/manageFile/ManageFile.jsx (hunk 1 above)
Comment (mubeenmughal) [reply src/components/newSidebar/manageFile/ManageFile.jsx]: resolved

File: src/components/newSidebar/reports/Reports.jsx (hunk 2 above)
Comment (mubeenmughal) [reply src/components/newSidebar/reports/Reports.jsx]: resolved

File: src/pages/Folders/modals/FoldersModal.jsx (hunk 3 above)
Comment (mubeenmughal) [reply src/pages/Folders/modals/FoldersModal.jsx:290]: resolve

**Pull request merged**